### ISSUE TITLE
Spyder crash after a few seconds

### ISSUE THREAD

**Issue by Mark531** [spyder-ide/spyder#17239]
### Issue Report Checklist

* [X] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [X] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [X] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [X] Tried basic troubleshooting (if a bug/error)
    * [X] Restarted Spyder
    * [X] Reset preferences with ``spyder --reset``
    * [X] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [X] Tried the other applicable steps from the Troubleshooting Guide
* [X] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
I create a new conda environment for python 3.6 and installed spyder. Running it causes a crash after a few seconds.

### What steps reproduce the problem?

1. conda create -n py36 python=3.6
2. conda activate py36
3. conda install -c anaconda spyder
4. Running Spyder (py36) from the start menu shortcuts

### What is the expected output? What do you see instead?
The Spyder window opens, then after about 5 seconds, it crashes.

### Paste Traceback/Error Below (if applicable)

Spyder crashes before I have access to its menu.

## Versions

* Spyder version: 4.1.5
* Python version: 3.6.13
* Qt version: 5.6.2
* PyQt version: 5.6.0
* Operating System name/version: Windows 10

### Dependencies

Spyder crashes before I have access to its menu.

**Comment by steff456** [comment]
Hi @Mark531,

Can you please try to install spyder using the conda-forge channel and see if the error persists?

```
conda create -n py36 -c conda-forge python=3.6
conda activate py36
conda install -c conda-forge spyder
Running Spyder (py36) from the start menu shortcuts
```

**Comment by Mark531** [comment]
Hi @steff456, I ran the command, but it only updated 3 packages:
```
The following packages will be UPDATED:

  ca-certificates    anaconda::ca-certificates-2020.10.14-0 --> conda-forge::ca-certificates-2021.10.8-h5b45459_0
  certifi            anaconda/win-64::certifi-2020.6.20-py~ --> pkgs/main/noarch::certifi-2020.6.20-pyhd3eb1b0_3
  openssl            anaconda::openssl-1.0.2t-vc14h62dcd97~ --> conda-forge::openssl-1.0.2u-hfa6e2cd_0
```
Upon installation, the problem persists. Is there a way to see logs?

**Comment by mmagnuski** [comment]
@Mark531 Could you try running the commands that @steff456 proposed but change the new environment name to, for example, `py36_2`? This way you are sure to create a new environment, not update existing one.

**Comment by jaxankey** [comment]
Not sure if this is related, but I pip-installed spyder on a fresh miniconda3 python 3.9 (windows 10), and running spyder.exe directly doesn't work (just quits quickly), but typing `spyder` from the anaconda prompt (which launches the same executable) does work, even after I close the console.

Any thoughts?

**Comment by steff456** [comment]
> Upon installation, the problem persists. Is there a way to see logs?

Can you try it in a fresh environment as @mmagnuski already mentioned?

> Any thoughts?

That is a weird behavior, could you try to install Spyder using conda from the conda-forge channel and see if the error persists?

**Comment by Mark531** [comment]
So, I created a new environment but "conda install" keeps hanging forever...
```
(py36) C:\Users\kguimard>conda install -c conda-forge spyder
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: |
```

**Comment by ccordoba12** [comment]
@Mark531, Python 3.6 support in conda-forge is no longer available and Spyder will drop support for it on its next version (5.3.0).

So, unless you really require running your code on that version, please update to Python 3.7. For that, you can use the following commands instead:

    conda create -n py37 -c conda-forge python=3.7
    conda activate py37
    conda install -c conda-forge spyder
    Running Spyder (py37) from the start menu shortcuts

**Comment by Mark531** [comment]
@ccordoba12 I see, but the project I'm working on uses python 3.6. I may run into incompatibilities by upgrading the version of python (I especially think of pandas and all its sub-components).

**Comment by mmagnuski** [comment]
@Mark531 You can setup spyder in one conda env and point it to use another one. See [this FAQ](https://docs.spyder-ide.org/current/faq.html#using-existing-environment).

**Comment by Mark531** [comment]
You're right! But for an unknown reason, I cannot select python.exe from my python 3.6 env, I get an error "Invalid file path".
The path I use is: C:/Users/<my_user>/Anaconda3/envs/py36/python.exe

**Comment by mmagnuski** [comment]
@Mark531 I don't know if the type of separator used will matter in this case, but you can try changing `/` to `\` (I'm on Windows and I see `\` separators in the accepted path).

**Comment by mmagnuski** [comment]
@Mark531 I've been through many of the same problems when setting spyder python interpreter to a different env. :)  Your "interpreter" environment requires a specific version of `spyder‑kernels`. The exact version depends on Spyder version you installed, but in your case you'd have to:
1. activate the "interpreter" conda env
2. `conda install spyder-kernels=2.1.* -c conda-forge`

**Comment by Mark531** [comment]
@mmagnuski I've installed spyder-kernels-2.1.3 as you advised on the py36 env. Then I open the Spyder version linked to that env. I had a crash at start-up (that I reported on github via the automatic procedure) but Spyder keeps running and seems to work... So, the initial issue seems to be linked with spyder-kernels, as if conda install hadn't installed it properly...
